third-party/realdds/include/realdds/dds-participant.h

```diff
@@ -50,21 +52,25 @@ class dds_participant
 
     struct listener_impl;
 
+    nlohmann::json _settings;
+
 public:
     dds_participant() = default;
     dds_participant( const dds_participant & ) = delete;
     ~dds_participant();
 
     // Creates the underlying DDS participant and sets the QoS
     // If need to use callbacks set them before calling init, they may be called before init returns.
-    void init( dds_domain_id, std::string const & participant_name );
+    void init( dds_domain_id, std::string const & participant_name, nlohmann::json const & );
```

Comment:
Add `settings` as parameter name to clarify the meaning.

Reply:
Done